oeps/oep-0005.rst

```diff
@@ -0,0 +1,182 @@
+=======================================
+OEP-5: Pre-built Developer Environments
+=======================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-5 </oeps/oep-0005>`             |
++---------------+-------------------------------------------+
+| Title         | Pre-built Developer Environments          |
++---------------+-------------------------------------------+
+| Last Modified | 2016-06-28                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+|  Created      | 2016-06-28                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Open edX is a large ecosystem of smaller services. In order to facilitate
+newcomers working in that system, each service will provide a prebuilt environment
+that is packaged, published, and ready to be used for development.
+
+Motivation
+==========
+
+Open edX and the edx-platform repository have gone through several iterations
+of developer setup instructions.
+
+The first set of instructions detailed how to install edx-platform and all of
```

Comment:
I think having it in there, as a way of describing the things that haven't worked in the past, is useful.